deploy/runtools/runtime_config.py

```diff
@@ -102,8 +94,8 @@ def get_boot_simulation_command(self, slotid, all_macs,
         runtime parameters currently. """
 
         # TODO: supernode support
-        tracefile = "+tracefile0=TRACEFILE0" if trace_enable else ""
-        autocounterfile = "+autocounter-filename0=AUTOCOUNTERFILE0"
+        tracefile = "+tracefile=TRACEFILE" if trace_enable else ""
```

Comment:
Naming needs to be updated in docs/Advanced-Usage/Debugging-and-Profiling-on-FPGA/TracerV.rst